src/main/java/apoc/load/util/LoadJdbcConfig.java

```diff
@@ -13,6 +13,8 @@
 
     private ZoneId zoneId = null;
 
+    private Map<String,Object> credentials;
```

Comment:
Non si può mettere come di seguito?
```suggestion
    private Map<String, String> credentials;
```
e togliere i vari `toString`